internal/generator/main.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"errors"
+	"log"
+	"os"
+	"strings"
+)
+
+var errUnmachedBraces = errors.New("braces are unmatched")
+
+func main() {
```

Comment:
I think here duplicate of `main`.

We need only `GenerateCode` in this file, and use it in `cmd/generator/main.go`.

_We needed to move `generateCode` to a separate package in order to simplify `cmd` and reuse this function in the future._